Libplanet.Tests/Tx/TxFixture.cs

```diff
@@ -42,15 +59,19 @@ public TxFixture()
                 },
             };
             TxWithActions = Transaction<PolymorphicAction<BaseAction>>.Create(
-                PrivateKey,
+                PrivateKey1,
                 actions,
                 timestamp: timestamp
             );
         }
 
-        public PrivateKey PrivateKey { get; }
+        public PrivateKey PrivateKey1 { get; }
+
+        public PrivateKey PrivateKey2 { get; }
+
+        public PrivateKey PrivateKey3 { get; }
 
-        public PublicKey PublicKey => PrivateKey.PublicKey;
+        public PublicKey PublicKey => PrivateKey1.PublicKey;
```

Comment:
To be consistent, we'd better append “1” to this property too.

```suggestion
        public PublicKey PublicKey1 => PrivateKey1.PublicKey;
```

Reply:
Fixed. (cf5eedbe757b312358635dec47e565ae0b46e671)